tutorials/vue/vue-websocket.md

```diff
@@ -0,0 +1,45 @@
+[WebSockets](/tutorials/node/websockets) are a great tool for when you want to show real time changes in data.
+For example, a server can push stock market price changes to the client rather than the client needing to ask for the changes via a http request, or in chat applications which reuse the WebSocket connection for the current users.
+With this being said, below you will find an example of a simple Vue application that shows the current time to the user and where
+the user can send a simple message to the websocket.
+
+```javascript
+[require:Vue vue-websocket]
+```
+
+The server code is:
+
+```javascript
+"use strict";
+
+const serverPort = 3000,
+  http = require("http"),
+  express = require("express"),
+  app = express(),
+  server = http.createServer(app),
+  WebSocket = require("ws"),
+  websocketServer = new WebSocket.Server({ server });
+
+//when a websocket connection is established
+websocketServer.on("connection", (webSocketClient) => {
+  // send feedback to the incoming connection
+  webSocketClient.send("The time is: ");
+  //when a message is received
+  webSocketClient.on("message", (message) => {
+    console.log(message);
+    //for each websocket client
+    websocketServer.clients.forEach((client) => {
```

Comment:
This is a bit nasty: every time the client sends a message, we create a new interval that sends a message every second. I think you should just get rid of the `webSocketClient.on('message')` wrapper, and just create a new interval in the `on('connection')` handler